src/main/java/com/avioconsulting/mule/connector/akv/provider/client/AzureClient.java

```diff
@@ -126,4 +129,22 @@ public boolean isValid() {
       return true;
     }
   }
+
+  public String getHttpBaseUri(){
+    String url = System.getProperty("AKV_TEST_URL");
+    if(url != null){
+      LOGGER.warn("Using AKV Test URL: " + url);
+      return url;
+    }
+    return "https://" + vaultName + AZURE_HOST ;
+  }
+
+  public String getAuthBaseUri(){
+    String url = System.getProperty("AKV_TEST_URL");
+    if(url != null){
+      LOGGER.warn("Using AKV Test URL: " + url);
+      return url;
+    }
```

Comment:
Similar to above, check for empty string.  Auth test URL will need to be different from the vault base url though so a different system property should be used.

Reply:
Not for mocked unit tests... However, thinking ahead to integration tests, we would potentially want to use that property, so a different property wouldn't be bad.

Reply:
added